app/views/ncr/work_orders/form.html.erb

```diff
@@ -98,6 +98,17 @@
                           disabled: approver_email_frozen?,
                           data: {attr: 'email_address', src: '/api/v1/users.json'} %>
     </div>
+    <% if @model_instance.new_record? %>
```

Comment:
Would prefer to not have two UIs for uploading, for usability and maintainability purposes. We should consider doing it all through the form, and getting rid of the Upload button on the `show` page.